io.github.baldeuniversel.am-okay.yml

```diff
@@ -0,0 +1,76 @@
+id: io.github.baldeuniversel.am-okay
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk 
+
+command: am-okay
+
+finish-args:
+    - --filesystem=host
+    - --filesystem=/tmp
```

Comment:
Why do you need host and /tmp? This is pretty extensive.

Reply:
Yep, the `am-okay` program partly imitates the Linux `cp` and `mv` commands. 

This is a sine quo non condition for this program to work correctly.

Reply:
This doesn't tell why you need tmp...

Reply:
still it doesn't tell. There is a `/tmp` in the sandbox.